From 67242e15c298abff4f93a7bfcd72372f7ba60270 Mon Sep 17 00:00:00 2001
From: Jan Friesse <jfriesse@redhat.com>
Date: Wed, 18 Nov 2015 15:45:19 +0100
Subject: [PATCH] Revert "ipcc: Fix ERR_LIBRARY error if finalise called inside dispatch"

This reverts commit 83bf77e6f42c2a8b01b1d079e8dc792d9cc7d35d.

Messages may be lost.

Signed-off-by: Jan Friesse <jfriesse@redhat.com>
---
 lib/coroipcc.c |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/lib/coroipcc.c b/lib/coroipcc.c
index 5aff6e0..774e835 100644
--- a/lib/coroipcc.c
+++ b/lib/coroipcc.c
@@ -957,13 +957,6 @@ coroipcc_dispatch_put (hdb_handle_t handle)
 retry_ipc_sem_wait:
 	res = ipc_sem_wait (ipc_instance->control_buffer, SEMAPHORE_DISPATCH, ipc_instance->fd);
 	if (res != CS_OK) {
-	        /*
-		 * IPC was closed earlier in dispatch, return OK
-		 */
-	        if (ipc_instance->control_buffer->ipc_closed) {
-		        res = CS_OK;
-			goto error_exit;
-		}
 		if (res == CS_ERR_TRY_AGAIN) {
 			priv_change_send (ipc_instance);
 			goto retry_ipc_sem_wait;
-- 
1.7.1

